src/dragging/dragging.js

```diff
@@ -22,7 +22,22 @@
     let onMouseMove = function(e) {
       let deltaX = e.clientX - clientX + startX;
       let deltaY = e.clientY - clientY + startY;
-      onMove(deltaX, deltaY);
+      onMove(deltaX, deltaY, e);
+      var dragTargets = document.elementsFromPoint(e.clientX, e.clientY);
```

Comment:
Trying to decide the drag targets on every `mousemove` makes me think of two potential problems (maybe they are already addressed by code elsewhere):

1) It is potentially expensive.
2) It is possible that there will be an error if the mouse moves enough in one tick to no longer be hovering over the element that it was previously dragging in the last tick.

Would it be possible to make this more stateful, so that a drag "starts" and "ends", and in between that time the `dragTargets` only need to be picked once?